db/documents.py

```diff
@@ -66,6 +91,8 @@ class Document(sqla.Model):
     validation_time = Column(sqla.DateTime(timezone=True), nullable=True)
     submitted_by = Column(sqla.String, nullable=True)
     legacy_id = Column(sqla.Integer, nullable=True)  # old id from fs-deluxe, so we can recognize the old barcodes
+    early_document_state = Column(sqla.Integer, nullable=False, server_default='0')
+    deposit_return_state = Column(sqla.Integer, nullable=False, server_default='0')
```

Comment:
The disbursal of early document reward and deposit are two separate actions. We want to delete the submitted_by name as soon as both actions cannot be performed anymore. Therefore the information, whether one of these actions can still be performed, is required.